DVR/HTTPURLResponse.swift

```diff
@@ -1,18 +1,18 @@
 import Foundation
 
-// There isn't a mutable NSHTTPURLResponse, so we have to make our own.
-class URLHTTPResponse: NSHTTPURLResponse {
+// There isn't a mutable HTTPURLResponse, so we have to make our own.
+class HTTPURLResponse: Foundation.HTTPURLResponse {
```

Comment:
Naming this class `HTTPURLResponse` requires us to use `Foundation.HTTPURLResponse` in a lot of places here and could be confusing. What do you think about `MutableHTTPURLResponse` since it's a mutable version of `HTTPURLResponse`?